sample_agents/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Sample agents level init."""
+from sample_agents.mathematics_agent import MathAgent
```

Comment:
Add an issue and a TODO here for removing MathAgent

Reply:
Should the issue include the removal of the class MathUser too?

Reply:
Yes please.

Reply:
Issue #162